CHANGELOG.md

```diff
@@ -34,8 +40,6 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 - Added more compiler configurations to GitLab CI.
```

Comment:
This is in the wrong place.

Reply:
Yeah.  It looks like it has been in the wrong place even since 4.1.1.  Strange

Reply:
It wasn't in 4.1.1